iot-identity-service.diff

```diff
@@ -0,0 +1,890 @@
+diff --git a/tpm/aziot-tpm-sys/azure-iot-hsm-c/CMakeLists.txt b/tpm/aziot-tpm-sys/azure-iot-hsm-c/CMakeLists.txt
```

Comment:
Is this a custom patch you created, or something picked up from somewhere?
Would be useful to have a header with a description in the patch file.

Reply:
This is a patch created from old code that cshari committed to cshari-zededa/lfedge-eve-tools/azure-on-eve
I took the same code, made some compatibility changes and created a patch that can be applied to azure/iot-identity-service. Will add comments.

Reply:
Also, I guess we can delete the azure-on-eve directory now.

Reply:
done